pwa/app/routes/event.$eventKey.tsx

```diff
@@ -299,3 +320,53 @@ function AwardsTab({ awards }: { awards: Award[] }) {
     </div>
   );
 }
+
+function TeamsTab({ teams }: { teams: Team[] }) {
+  teams.sort(sortTeamsComparator);
+
+  // Lot todo here:
+  // 1. add robot images
+  // 2. create a TeamEvent component that shows schedule, etc, which is inside CredenzaBody
+  // 3. add hover effects
+  return (
+    <div className="md:columns-2">
+      {teams.map((t) => (
+        <Credenza key={t.key}>
```

Comment:
Would we need to do this every time we have a link to a team (e.g. for every match on an event page)? I feel like this would be more scalable if we tied the modal to the route and rely on link navigation to trigger the modals.

Reply:
It depends on what you want. I personally am not really sure I'd want this modal/drawer when clicking from say rankings? But we could try it?

In any case, we would be altering `CredenzaTrigger` children, while the content stays the same. IMO splitting this out into reusable TBA-specific components can be in a future PR, but I can put it in the todo list I have there.